vshard/router/init.lua

```diff
@@ -710,6 +718,10 @@ local function router_callbre(router, bucket_id, ...)
     return router_call_impl(router, bucket_id, 'read', true, true, ...)
 end
 
+local function router_callbzre(router, bucket_id, ...)
+    return router_call_impl(router, bucket_id, 'read', true, 2, ...)
```

Comment:
If we want to introduce such feature, we should probably make API more explicit. Using `2` as the value for `balance` doesn't seem too good. A suppose, the better solution would be to make `balance` a string, and if its value is something like `prefer_zone`, balance replicas in a way, you propose. In all other cases of `balance`, simple rr can be used